scripts/zones/Ceizak_Battlegrounds/mobs/Taxet.lua

```diff
@@ -0,0 +1,80 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Tax'et
+-----------------------------------
```

Comment:
This will also need comments with commands for `!pos` (take you to the NM's spawn point), and `!additem` or some sort of `!exec` to force-pop the NM, for easier testing